<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGameDialog</class>
 <widget class="QDialog" name="AddGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="descriptionEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>104</width>
     <height>64</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="genreEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="platformEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ratingEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="releaseDateEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="developerEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="countryEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minOsEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minProcessorEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minRamEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minGraphicsEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minDirectXEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minNetworkEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minDiskSpaceEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minAdditionalEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recGraphicsEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recDiskSpaceEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recOsEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recAdditionalEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recRamEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recProcessorEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recDirectXEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recNetworkEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="videoIdEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>450</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="imagePathEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
